zipline/data/loader.py

```diff
@@ -182,19 +182,13 @@ def load_market_data(bm_symbol='^GSPC'):
     # we have data,then we need to update
     if len(days_up_to_now) - last_bm_date_offset > 1:
         benchmark_returns = update_benchmarks(bm_symbol, last_bm_date)
-        if (
-            benchmark_returns.index.tz is None
-            or
-            benchmark_returns.index.tz.zone != 'UTC'
-        ):
+        if benchmark_returns.index.tz is None or \
```

Comment:
Since this is outside the bounds of PEP8, it's a matter of stylistic opinion which is left to the discretion of the developer. I prefer it this way, and I did the work. So shut yer trap. ;-)